public/documentation.html

```diff
@@ -40,11 +40,11 @@ <h1>Documentation for PHPUnit</h1>
      <ul>
       <li><a href="https://phpunit.readthedocs.io/en/latest/">English Documentation for PHPUnit</a></li>
       <li><a href="https://phpunit.readthedocs.io/fr/latest/">French Documentation for PHPUnit</a></li>
-      <li><a href="https://phpunit.readthedocs.io/pt_br/latest/">Brazilian Portuguese Documentation for PHPUnit</a></li>
+      <li><a href="https://phpunit.readthedocs.io/pt_BR/latest/">Brazilian Portuguese Documentation for PHPUnit</a></li>
       <li><a href="https://phpunit.readthedocs.io/ja/latest/">Japanese Documentation for PHPUnit</a></li>
       <li><a href="https://phpunit.readthedocs.io/zh_CN/latest/">Simplified Chinese Documentation for PHPUnit</a></li>
      </ul>
-     <p>Documentation for PHPUnit 6.5 (and older) is archived at <code>https://phpunit.de/manual/6.5/en/index.html</code>. Simply replace <code>6.5</code> with the version number you are looking for. Simply replace <code>en</code> with <code>fr</code>, <code>pt_br</code>, <code>ja</code>, or <code>zh_cn</code> to access the French, Brazilian Portuguese, Japanese, or Simplified Chinese translation, respectively.</p>
+     <p>Documentation for PHPUnit 6.5 (and older) is archived at <code>https://phpunit.de/manual/6.5/en/index.html</code>. Simply replace <code>6.5</code> with the version number you are looking for. Simply replace <code>en</code> with <code>fr</code>, <code>pt_BR</code>, <code>ja</code>, or <code>zh_cn</code> to access the French, Brazilian Portuguese, Japanese, or Simplified Chinese translation, respectively.</p>
```

Comment:
This is wrong: `https://phpunit.de/manual/6.5/pt_BR/index.html` gives a 404, `https://phpunit.de/manual/6.5/pt_br/index.html` is correct.

Reply:
Ow yes.. sorry.. I wanted to fix only the readthedocs link 😄